modules/aws-exfiltration-protection/versions.tf

```diff
@@ -3,12 +3,12 @@ terraform {
   required_providers {
     databricks = {
       source  = "databricks/databricks"
-      version = ">=0.5.1"
+      version = ">=1.27.0"
     }
 
     aws = {
       source  = "hashicorp/aws"
-      version = "~> 4.58.0"
+      version = "~> 5.20.1"
```

Comment:
We need to remove versions from the modules

Reply:
I followed all other examples. Versions are included in all of them.